scripts/contentScript.js

```diff
@@ -666,7 +768,75 @@ const orchestrator = (tabUrl, websiteUrl, userId) => {
 				}
 			}
 		}
-	})
+
+		if (urlObj[2] === 'gitlab.com') {
+
+			if(urlObj.length === 4 && !urlObj[3]){
+				//TODO: need repo_owner from userID
+				
+				
+
+			}
+
+			else if (urlObj.length > 4){	
+
+				const ownerName = urlObj[3];
+				const repoName = urlObj[4];
+				showFloatingActionButton(ownerName, repoName, userId, websiteUrl, 'gitlab');
+				
+				if (urlObj[5] && (urlObj[6] === 'merge_requests') && !urlObj[7]){
+					console.log("Showing MRs");
+					addingCssElementToGitLab(1,2,3);
+
+					const body = {
+						"repo_owner": ownerName,
+						"repo_name": repoName,
+						"user_id": userId,
+						"repo_provider": 'gitlab' 
+					}
+
+					const url = `${backendUrl}/relevance/pr`;
+					const highlightedMRIds = await apiCall(url, body);
+					highlightRelevantMRs(highlightedMRIds); // TODO: build this function
+				}
+				if(urlObj[5] && (urlObj[6] === 'merge_requests') && urlObj[7])
+				{
+					console.log("Viewing particular MR")
+					const prNumber = parseInt(urlObj[7]);
+					const body = {
+						"repo_owner": ownerName,
+						"repo_name": repoName,
+						"user_id": userId,
+						"pr_number": prNumber,
+						"repo_provider": 'gitlab',
+					}
+					const url = `${backendUrl}/relevance/pr/files`;
+
+					console.log("Calling API");
+
+					if(env!=='dev')
```

Comment:
Why is this necessary? Your URL will already be modified based on whether you are working in dev environment or prod.